core/modules/widgets/navigator.js

```diff
@@ -277,6 +277,8 @@ NavigatorWidget.prototype.makeDraftTiddler = function(targetTitle) {
 	}
 	// Get the current value of the tiddler we're editing
 	var tiddler = this.wiki.getTiddler(targetTitle);
+	var defaultType = this.wiki.getTiddler("$:/config/DefaultMissingType");
```

Comment:
`this.wiki.getTiddlerText(title,defaultText)` would allow you to define the default type, if the config tiddler does not exist. 
For situations like this we do define a default value.
See. https://github.com/TiddlyWiki/TiddlyWiki5/blob/dabbc1bacc35b3981959065f88adbb3b7ec1c2ff/core/modules/wiki.js#L1499

Reply:
Updated to use `getTiddlerText` with default.